ci/ray_ci/automation/filter_flaky_tests.py

```diff
@@ -1,29 +1,32 @@
 import sys
 
-from ci.ray_ci.utils import filter_out_flaky_tests
+from ci.ray_ci.utils import filter_flaky_tests
 from ray_release.configs.global_config import init_global_config
 from ray_release.bazel import bazel_runfile
 
 
 def main():
     """
-    Filter out flaky tests.
+    Filter flaky tests.
     Input (stdin) as a list of test names.
     Output (stdout) as a list of test names without flaky tests.
 
     Args:
         prefix: Prefix to query tests with.
+        select_flaky: If true, only print flaky tests.
+            If false, only print non-flaky tests.
     """
     # Process arguments
-    if len(sys.argv) != 2:
+    if len(sys.argv) != 3:
         raise ValueError("Invalid number of arguments.")
 
     prefix = sys.argv[1]
+    select_flaky = sys.argv[2]
```

Comment:
this is a string, but you are sending in as a boolean.

you would need to parse it some how.

could you use the `click` package for command line parsing, which is used in other command line tools in `ray_ci`?

Reply:
fixed!